### PR TITLE
adds image loading state

### PR DESCRIPTION
**Pull request by bladicito**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -16,6 +17,7 @@ export default async function Page() {
       <div className="content-bottom max-w-4xl mx-auto">
         {session && (
           <div className="">
```
Comment (lehmi11) [inline app/page.tsx:19]: `className` is not used and can be deleted

File: components/placeholders/PostTextPlaceholder.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { PostHeaderPlaceholder } from '@/components/placeholders/PostHeader';
+
+export const PostTextPlaceholder = ({ lines }: { lines: number }) => {
```
Comment (lehmi11) [inline components/placeholders/PostTextPlaceholder.tsx:3]: we might want to define the props in a separate IProps interface as we did in other components. Just for consistency. What do you think of an default value in this component? would that make sense?

File: components/post-image/PostImage.tsx [hunk 3]
```diff
@@ -2,32 +2,46 @@
...
-          <Icon type={EIConTypes.FULL_SCREEN} fitParent={true} />{' '}
+    <div className="relative">
+      <div
+        className={`${loaded ? '-z-10 opacity-0' : 'z-10 opacity-100'} absolute top-0 left-0 right-0 bottom-0 ${imageProportions} ${imageTransition}`}
+      >
+        <PostImagePlaceholder />
+      </div>
+      <div
+        onClick={() => {
+          alert('to be open in an overlay modal or lightbox');
```
Comment (lehmi11) [inline components/post-image/PostImage.tsx:25]: Is this the final implementation? If not, we could perhaps add a TODO-comment

File: components/post-image/PostImage.tsx [hunk 4]
```diff
@@ -2,32 +2,46 @@
...
+        }}
+        className={`${loaded ? 'opacity-100' : 'opacity-0'} rounded-2xl relative h-0 overflow-hidden group cursor-pointer ${imageProportions} ${imageTransition}`}
+      >
+        <div className="group-hover:opacity-70 bg-violet-600 top-0 left-0 right-0 bottom-0 opacity-0 z-30 absolute transition duration-500 flex items-center justify-center">
+          <div className="group-hover:w-12 group-hover:h-12 w-2 h-2 text-white transition[width] duration-500 group-hover:opacity-90 opacity-0">
+            <Icon type={EIConTypes.FULL_SCREEN} fitParent={true} />{' '}
```
Comment (lehmi11) [inline components/post-image/PostImage.tsx:31]: is `{' '}` necessary?

**Review by lehmi11 — APPROVED** [review]
LGTM. some minor things

File: components/placeholders/PostTextPlaceholder.tsx (hunk 2 above)
Comment (bladicito) [reply components/placeholders/PostTextPlaceholder.tsx:3]: we can add like 3 lines as per default 👍🏽

File: components/post-image/PostImage.tsx (hunk 3 above)
Comment (bladicito) [reply components/post-image/PostImage.tsx:25]: hahaha. no. On the Modal PR the content of the image will be set into a modal.

File: components/post-image/PostImage.tsx (hunk 4 above)
Comment (bladicito) [reply components/post-image/PostImage.tsx:31]: nop.

**Pull request merged**